packages/Button/src/Button.styles.js

```diff
@@ -15,23 +14,23 @@ const enabled = content => css`
 // Borders
 
 const borderColors = {
-  [Kinds.DEFAULT]: tokens.border.color,
-  [Kinds.PRIMARY]: tokens.color.green,
-  [Kinds.SECONDARY]: tokens.color.purple,
-  [Kinds.DESTRUCTIVE]: tokens.color.orange,
-  [Kinds.FLAT]: tokens.border.color,
-  [Kinds.MINOR]: "transparent",
-  [Kinds.LINK]: "transparent",
+  [types.DEFAULT]: tokens.border.color,
+  [types.PRIMARY]: tokens.color.green,
+  [types.SECONDARY]: tokens.color.purple,
+  [types.DESTRUCTIVE]: tokens.color.orange,
+  [types.FLAT]: tokens.border.color,
+  [types.MINOR]: "transparent",
+  [types.LINK]: "transparent",
 };
 
 const borderHoverColors = {
-  [Kinds.DEFAULT]: tokens.border.hoverColor,
-  [Kinds.PRIMARY]: tokens.color.greenDarken10,
-  [Kinds.SECONDARY]: tokens.color.purpleDarken10,
-  [Kinds.DESTRUCTIVE]: tokens.color.orangeDarken10,
-  [Kinds.FLAT]: tokens.border.hoverColor,
-  [Kinds.MINOR]: "transparent",
-  [Kinds.LINK]: "transparent",
+  [types.DEFAULT]: tokens.border.hoverColor,
+  [types.PRIMARY]: tokens.color.greenDarken10,
+  [types.SECONDARY]: tokens.color.purpleDarken10,
+  [types.DESTRUCTIVE]: tokens.color.orangeDarken10,
+  [types.FLAT]: tokens.border.hoverColor,
+  [types.MINOR]: "transparent",
+  [types.LINK]: "transparent",
```

Comment:
Just wondering why not using `Button.types.kind.LINK` here? I saw inside side panel component you have `[Button.types.kind.PRIMARY]` in `styles.js` .  Maybe you don't need to change them right away but I think we should revisit and make them consistent.